### PR TITLE
Tekton Pipeline and Triggers

### PR DESCRIPTION
**Pull request by SM43**
Tekton Pipeline for frontend - have 2 tasks - 
1. buildah - create an image from git and push it to OpenShift internal registry
2. oc apply deployment - create a deployment for the image

Tekton Trigger - which listens to github push and pull request merge event and trigger the pipeline

### CODE REVIEW

**Comment by invincibleJai** [comment]
@SM43 some description would be good to have along with a checklist, what all this PR targets

**Comment by SM43** [comment]
> @SM43 some description would be good to have along with a checklist, what all this PR targets
Working on the documentation will update the readme soon.

File: pipeline/06-github-webhooks/03_create-webhook-task.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    args:
+    - -ce
+    - |
+      set -e
+      echo "Create Webhook"
+      if [ $(inputs.params.GitHubDomain) = "github.com" ];then
+        curl -v -d "{\"name\": \"web\",\"active\": true,\"events\": $(inputs.params.WebhookEvents),\"config\": {\"url\": \"$(inputs.params.ExternalDomain)\",\"content_type\": \"json\",\"insecure_ssl\": \"1\" ,\"secret\": \"$(cat /var/secret/$(inputs.params.GitHubSecretStringKey))\"}}" -X POST -u $(inputs.params.GitHubUser):$(cat /var/secret/$(inputs.params.GitHubAccessTokenKey)) -L https://api.github.com/repos/$(inputs.params.GitHubOrg)/$(inputs.params.GitHubRepo)/hooks
+      else
+        curl -d "{\"name\": \"web\",\"active\": true,\"events\": $(inputs.params.WebhookEvents),\"config\": {\"url\": \"$(inputs.params.ExternalDomain)/\",\"content_type\": \"json\",\"insecure_ssl\": \"1\" ,\"secret\": \"$(cat /var/secret/$(inputs.params.GitHubSecretStringKey))\"}}" -X POST -u $(inputs.params.GitHubUser):$(cat /var/secret/$(inputs.params.GitHubAccessTokenKey)) -L https://$(inputs.params.GitHubDomain)/api/v3/repos/$(inputs.params.GitHubOrg)/$(inputs.params.GitHubRepo)/hooks
+      fi
```
Comment (invincibleJai) [inline pipeline/06-github-webhooks/03_create-webhook-task.yaml]: new line pls

Comment (SM43) [reply pipeline/06-github-webhooks/03_create-webhook-task.yaml]: added. will push the change in next PR.

**Pull request merged**